librz/bin/format/elf/glibc_elf.h

```diff
@@ -2712,6 +2712,107 @@ enum {
 #define SHT_ARM_PREEMPTMAP (SHT_LOPROC + 2) /* Preemption details.  */
 #define SHT_ARM_ATTRIBUTES (SHT_LOPROC + 3) /* ARM attributes section.  */
 
+/* Constants defining public attributes sub-subsections.  */
+#define ARM_TAG_FILE    1
+#define ARM_TAG_SECTION 2
+#define ARM_TAG_SYMBOL  3
+
+/* ARM target-related attributes.  */
+#define TAG_CPU_RAW_NAME         4
+#define TAG_CPU_NAME             5
+#define TAG_CPU_ARCH             6
+#define Tag_CPU_ARCH_PROFILE     7
```

Comment:
```suggestion
#define TAG_CPU_ARCH_PROFILE     7
```

Reply:
Nice catch! Copypasta formatting error. I'll fix this.